teachopencadd/talktorials/018_md_simulation/talktorial.ipynb

```diff
@@ -0,0 +1,1155 @@
+{
```

Comment:
<blockquote>In this talktorial, we use the pdb structure of this kinase in complex with the small molecule inhibitor 03P to perform an MD simulation (<a href="https://www.rcsb.org/structure/3poz" target="_blank">RCSB</a>).</blockquote><p>Since there exist several pdb structures for this kinase, it would be good to include the 4-letter code here: ... we use the pdb structure <strong>3poz</strong> ...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>